ledger/store/trackerdb/testsuite/accounts_kv_test.go

```diff
@@ -267,63 +270,107 @@ func CustomTestResourcesQueryAllLimited(t *customT) {
 	refAccB, err := aow.InsertAccount(addrB, accDataB.NormalizedOnlineBalance(t.proto), accDataB)
 	require.NoError(t, err)
 
-	// resource A-0 for accounts A and B
-	resDataA0 := trackerdb.ResourcesData{}
+	// asset A-0 for accounts A and B
+	resDataA0AcctA := trackerdb.ResourcesData{}
+	resDataA0AcctA.SetAssetHolding(basics.AssetHolding{Amount: 10})
+	resDataA0AcctA.SetAssetParams(basics.AssetParams{
+		Total: 100,
+	}, true)
+	// Non-creators will inherit asset params from the creator
+	resDataA0AcctB := trackerdb.ResourcesData{}
+	resDataA0AcctB.SetAssetHolding(basics.AssetHolding{Amount: 0})
 	aidxResA0 := basics.CreatableIndex(1)
-	_, err = aow.InsertResource(refAccA, aidxResA0, resDataA0)
+
+	_, err = aow.InsertResource(refAccA, aidxResA0, resDataA0AcctA)
 	require.NoError(t, err)
-	_, err = aow.InsertResource(refAccB, aidxResA0, resDataA0)
+	_, err = aow.InsertResource(refAccB, aidxResA0, resDataA0AcctB)
 	require.NoError(t, err)
 
-	// resource A-1 for accounts A and B
-	resDataA1 := trackerdb.ResourcesData{
-		Total: 10000,
-	}
+	// App A-1 for accounts A and B - this should be completely ignored
+	resDataA1AcctA := trackerdb.ResourcesData{}
+	appParams := ledgertesting.RandomAppParams()
+	resDataA1AcctA.SetAppParams(appParams, true)
+	resDataA1AcctB := trackerdb.ResourcesData{}
+	appParamsB := ledgertesting.RandomAppParams()
+	resDataA1AcctB.SetAppParams(appParamsB, true)
```

Comment:
This is more accurate for a single app that A creates and both accounts are opted into
```suggestion
	// App A-1 for accounts A and B - this should be completely ignored
	resDataA1AcctA := trackerdb.ResourcesData{}
	appParams := ledgertesting.RandomAppParams()
	resDataA1AcctA.SetAppParams(appParams, true)
	resDataA1AcctA.SetAppLocalState(basics.AppLocalState{})
	resDataA1AcctB := trackerdb.ResourcesData{}
	resDataA1AcctB.SetAppLocalState(basics.AppLocalState{})
```